README.md

```
@@ -1,3 +1,7 @@
+> **Phenotype Fork**: This is KooshaPari's fork of [isaacphi/mcp-language-server](https://github.com/isaacphi/mcp-language-server), used as the LSP backend for heliosCLI and AgilePlus code intelligence features in the Phenotype ecosystem.
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)